src/Tizen.NUI/src/public/BaseComponents/TextEditorBindableProperty.cs

```diff
@@ -32,21 +32,16 @@ public partial class TextEditor
         
         internal static void SetInternalTextProperty(BindableObject bindable, object oldValue, object newValue)
```

Comment:
Hello, 
I am curious about why this 'SetInternalTextProperty` has been remained even though this is not used.
is this still used in some other place?

Reply:
anyway, it can be removed later, so it would be OK for me.

Reply:
> Hello, I am curious about why this 'SetInternalTextProperty` has been remained even though this is not used. is this still used in some other place?

It is used when Xaml is enabled:
if (NUIApplication.IsUsingXaml)
{
    TextProperty = BindableProperty.Create(nameof(Text), typeof(string), typeof(TextEditor), string.Empty, propertyChanged: SetInternalTextProperty, defaultValueCreator: GetInternalTextProperty);
   ....